File: bookbeautysalon/bookBeauty.Services/Database/Transaction.cs
using System;
using System.Collections.Generic;

namespace bookBeauty.Services.Database;

public partial class Transaction
{
    public int TransactionId { get; set; }

    public int OrderId { get; set; }

    public float Price { get; set; }

    public required string Status { get; set; }

    public required string Name { get; set; }

    public virtual Order? Order { get; set; }
}

File: UI/bookbeauty_desktop/lib/models/transaction.dart
import 'package:json_annotation/json_annotation.dart';

part 'transaction.g.dart';

@JsonSerializable()
class Transaction {
  int? transactionId;
  String? name;
  int? orderId;
  double? price;
  String? status;

  Transaction(
      {this.transactionId, this.name, this.orderId, this.price, this.status});

  factory Transaction.fromJson(Map<String, dynamic> json) =>
      _$TransactionFromJson(json);

  Map<String, dynamic> toJson() => _$TransactionToJson(this);
}